tests/common/ixia/ixia_helpers.py

```diff
@@ -734,6 +733,33 @@ def create_pause_traffic(session, name, source, pkt_per_sec, pkt_count=None,
 
     return traffic_item
 
+def get_location (intf) :
+    """ Extracting location from interface, since TgenApi accepts location 
+    in terms of chassis ip, card, and port in different format.
+    
+    Note: Interface must have the keys 'ip', 'card_id' and 'port_id'
+                                                                                                           
+    Args:
+    intf (dict) : intf must containg the keys 'ip', 'card_id', 'port_id'.
+        Example format :
+        {'ip': u'10.36.78.53', 
+         'port_id': u'1', 
+         'card_id': u'9', 
+         'speed': 100000, 
+         'peer_port': u'Ethernet0'}
+
+    Returns: location in string format. Example: '10.36.78.5;1;2' where
+    1 is card_id and 2 is port_id.
+    """    
+    location = None
+    try :
+        location = str("%s;%s;%s"%(intf['ip'],intf['card_id'],intf['port_id']))
+    except :
+        pytest_assert(False,
+            "Interface must have the keys 'ip', 'card_id' and 'port_id'")
```

Comment:
Please use pytest.fail(msg) instead.

Reply:
I have replaced this pytest_assert(False, ...) with pytest.fail(msg)